client/containers/HomePage.js

```diff
@@ -18,12 +18,16 @@ class HomePage extends Component {
   }
   render() {
     const { condition, place } = this.props;
+    let btnStatus = true;
+    if (condition.latitude) {
+      btnStatus = false;
+    }
```

Comment:
why do you need to declare a let when it's only needed to be assigned once?

Reply:
and tests please

Reply:
i declare as let because the variable need re-assigned on the re-render of the conditional props.